docs/products/postgresql/howto/troubleshooting-connection-pooling.rst

```diff
@@ -0,0 +1,25 @@
+Troubleshooting connection pooling 
+==================================
+
+PgBouncer is a lightweight connection pooler for PostgreSQL® with low memory requirements (2 kB per connection by default).
+
+PgBouncer offers several methods when rotating connections:
+
+Session pooling
+  Most polite method. When a client connects, a server connection will be assigned to it for the whole duration it stays connected. When the client disconnects, the server connection will be put back into pool. This mode supports all PostgreSQL features.
```

Comment:
This is the most <please replace "polite" with something else, e.g. accommodating?> method. When a client connects, it gets assigned with a server connection that is maintained as long as the client stays connected. When the client disconnects, the server connection is put back into the pool. This mode supports all PostgreSQL features.